src/main/resources/db/migration/V1_70__referral_assignment_history.sql

```diff
@@ -0,0 +1,21 @@
+create table referral_assignments
+(
```

Comment:
Because there is no pk - should we have an index so it avoids a table scan?

Reply:
good idea. on referral_id ?

Reply:
done